puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/packaging.groovy

```diff
@@ -0,0 +1,61 @@
+def packages_to_build
+
+pipeline {
+    agent { label 'rpmbuild' }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        ansiColor('xterm')
+        buildDiscarder(logRotator(numToKeepStr: '15'))
+    }
```

Comment:
So, I've learned that these seem to not always work when deployed through JJB. We can keep them but let's keep an eye on it.

Reply:
we'll see how that goes I guess